services/src/abbr_query_matcher.py

```diff
@@ -0,0 +1,178 @@
+import pandas as pd
+import regex as re
+import jaro
+from constants import SearchConstants
+
+
+class AbbrQueryMatcher:
+
+    def __init__(self):
+        self.common_words_set = {'sec', 'st', 'sr', 'the', 'of', 'new', 'no'}
+
+
+    def filter_df_by_first_char(self, query: str, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
rename to filter_by_first_char as datatype can be referred by typing

Reply:
Renamed function to "filter by first char".